other/materials_designer/create_point_defect.ipynb

```diff
@@ -10,17 +10,10 @@
     "<h2 style=\"color:green\">Usage</h2>\n",
```

Comment:
<p>List numbering is broken</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/147/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Only on review, Not in the JL notebooks themselves